### DISCUSSION TITLE
Sapphire template-is it possible to increase/decrease text only on entire page?

### DISCUSSION THREAD

**Discussion by vitsin** [primefaces/community#1021] [PrimeFaces Templates]
Sapphire Layout: is it possible to increase/decrease the actual text of all page components without touching the size of components themselfs? 
Say, there is a <div class="card"> with h5="Monthly Sales" and another with h5="Total Views" etc, so is there a way to increase/decrease the size of fonts for "Monthly Sales"? And to make such change applicable to any page?

thanks

**Comment by tandraschko** [comment]
for now we decided to only have maintain the following templates:

          - "apollo"
          - "atlantis"
          - "avalon"
          - "diamond"
          - "freya"
          - "poseidon"
          - "ultima"
          - "verona"
